### ISSUE TITLE
Staging Review finding: Importance is communicated using color alone

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#106793]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** AEDP
**Product:** Design Patterns
**Feature:** Critical Information and Service List Item Components

## Findings details
**VA.gov Experience Standard - issue:** [User can perceive all meaningful elements.](https://design.va.gov/about/experience-standards/#user-can-perceive-all-meaningful-elements)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility), @kkitagawa-bosch (content)

### Description

The critical action component consists of a yellow background, left-aligned bold black text, and a right-aligned chevron icon. In the [accessibility considerations section](https://dev-design.va.gov/3973/components/critical-action#accessibility-considerations), the guidance says:
> WCAG indicates that we should not use color alone to convey information or distinguish a visual element. Although Critical Action has a yellow background, its critical nature is also signaled by:
> - Its action-oriented wording and date or deadline
> - Its position near the top of the card, service list item, or page
> - Its shaded background (regardless of what color the shading is perceived as) and dark chevron

I disagree with this assessment. Taking those points in reverse order...
- Shaded backgrounds are used for a variety of purposes on VA.gov, and do not consistently indicate that something is critical. Given the similarity in form to the primary action button (border radius, height, font size, font weight), I am not confident that a shaded background for a user who doesn't perceive it as bright yellow would be sufficient.
- Info alerts, success alerts, and other non-critical content are also routinely positioned at the top of a card, list item, or page.
- Action-oriented wording is a part of our content guidance for all [links](https://design.va.gov/content-style-guide/links#link-text) and [buttons](https://design.va.gov/content-style-guide/button-labels), regardless of their criticality.

But I'm not confident that the examples provided in the component guidance provide clear enough model for what constitutes "action-oriented wording [with a] date or deadline." The examples given are:
- "Add evidence by July 23, 2025"
- "We requested more information from you. Submit it by May 21, 2025."
- "Pay your debt of $251.42"
- "Check in to your Optometry appointment now"

The criticality of those examples is... debatable? It's unclear what criteria are sufficient for a "regular" link to graduate up to a critical action, beyond "we think this is important." They don't have a consistent structure, they don't all have deadlines (or even a temporal component), and there's nothing indicating what the potential consequences of not completing the task may be. I feel that lack of clarity opens the door for a lot of critical action links whose text doesn't in any way convey its importance.

Some additional thoughts around this: Currently, the alert components don't announce themselves to screen reader users with their type --- error, warning, success, info. There's no alt text on the icon that says "Warning" or "Info" etc. The thought process when this was discussed previously was that the content of the alert heading and body should be sufficient to communicate the semantics of the alert.

But there's been [some discussion based on user research](https://dsva.slack.com/archives/C0NGDDXME/p1737986432550589) that this might be the wrong approach --- that screen reader users might be missing out on an important cue that's communicated visually through the color and icon in the alert.

This question is on the list of research topics that the ADE team has compiled (cc @artsymartha68). I think affirmatively stating in our design system that content, position, and color are sufficient to meet WCAG 1.4.1 should be removed pending the results of that research, and I do not recommend moving forward with the component in its current form without either:
- Having something else that communicates criticality; OR
- Having clearer guidance around what the link text should be AND having user research that supports the idea that content alone is sufficient.

<details><summary>Link, screenshot or steps to recreate</summary>

![Critical action in context with other elements. The critical action consists of a yellow background with rounded corners, left-aligned black bold text, and a right-aligned chevron icon.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/105516-Brian-3.png?raw=true)

</details>

## Recommended action

I think there needs to be some kind of indicator that a critical action is actually critical independent of content, positioning, and color. Some options could include:
- A consistent prefix in the link text, as already occurs in the `aria-label` for screen reader users: `Action required: [link text]`.
- An icon associated with importance or warnings
- Supporting secondary text, eg. something like
   > **Add evidence by July 23, 2025**
   > Failure to do so may result in your claim being denied

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 1.4.1 Use of Colour - Level A](https://www.w3.org/TR/WCAG21/#use-of-color)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by msbtterswrth** [comment]
@rtwell we'd love to get your thoughts on this feedback when you've got some time.

**Comment by rtwell** [comment]
A few things here. 

1. FYSA feedback containing phrases like "I'm not confident…", "users might be…", I will disregard. This is why we test things. I want to know how things are failing (or not). 
2. Yes, some users have a lot of things going on, but that does not mean we shouldn't tell them the information they need to know. If there is a lot, then there is a lot, that is that particular user's situation, and I don't think we should pretend like it isn't. These things are prioritized at the top of the page **so users can resolve them** and ideally bring that number to zero. 
3. The feedback "The criticality of those examples is... debatable?" is moot because VA is not in the business of intuiting what is more important to a veteran—that is to say we cannot know which information we provide to veterans is more important than others given the subjective nature of each Veteran's situation. We **are** in the business of telling users when there are activities that are time sensitive, and when there are consequences to actions (or inaction). For example, "Your Rx for Medication X will expire on Date". Sometimes this is could be an oversight on the users part and they need to take immediate action because a lapse in medication can have negative effects. Conversely, "Your Rx for Medication X will expire on Date" might be 100% expected and irrelevant because it was a temporary medication that they were taking for 2 weeks (like an antibiotic). Another similar UX to this are how mobile operating systems deliver notifications — they are delivered in the order they are received, they do not prioritize notifications from App X over notifications from App Z because it cannot know. We default to assuming users are smart, and in control of their lives (or working with someone who is). They decide what is important. 
4. I do agree that the more consistent and thoughtful we are about the structure and delivery of the content for these, the better and if we need stronger guidance, then we should provide it — based on user feedback.
